pancancer-reporting/src/main/java/info/pancancer/arch3/reporting/ReportAPI.java

```diff
@@ -30,6 +30,10 @@
  */
 public interface ReportAPI {
 
+    public enum CloudTypes {
```

Comment:
Is it possible that this will be used outside the reporting component? If so, should this be moved outside of interface ReportAPI?